themes/digital.gov/src/scss/new/shortcodes/_cspan-iframe.scss

```diff
@@ -0,0 +1,14 @@
+.cspan-embed {
+  iframe {
+    aspect-ratio: 4/3;
+    height: 100%;
+    width: 100%;
+  }
+
+  // iframe classes to override to create perfectly fit video player
+  body.video.standalone section {
+    background-color: none;
+    margin-top: 0;
+    padding: 0; 
+  }
```

Comment:
Let's remove this if it doesn't work.

Reply:
Removed

Reply:
Removed in [6944183](https://github.com/GSA/digitalgov.gov/pull/5827/commits/6944183eeac8a0d3ca786fe71927a51e4f0866fe).